pkg/apis/config/feature_flags.go

```diff
@@ -80,8 +80,12 @@ const (
 	DefaultSendCloudEventsForRuns = false
 	// DefaultEmbeddedStatus is the default value for "embedded-status".
 	DefaultEmbeddedStatus = FullEmbeddedStatus
-	// DefaultEnableSpire is the default value for "enable-spire".
-	DefaultEnableSpire = false
+	// EnforceNonfalsifiabilityWithSpire is the value used for  "enable-nonfalsifiability" when SPIRE is used to enable non-falsifiability.
+	EnforceNonfalsifiabilityWithSpire = "spire"
+	// EnforceNonfalsifiabilityNone is the value used for  "enable-nonfalsifiability" when non-falsifiability is not enabled.
+	EnforceNonfalsifiabilityNone = ""
```

Comment:
why don't we consider Spire feature flag as a bool `EnforceNonfalsifiabilityWithSpire`? Will there be any other choices?

Reply:
Not sure, but that was what was decided in the TEP-0089. 
There are other ways to enforce non-falsifiable provenance. As of now there is only SPIRE based way.